src/oc/ocWrapper.ts

```diff
@@ -675,6 +675,13 @@ export class Oc {
             .catch((error) => namespaces);
     }
 
+    public async hasImageInfo(url: string): Promise<boolean> {
+        const result = await CliChannel.getInstance().executeTool(
```

Comment:
Try this with `docker.io/library/mongo`. For me at least, it fails, so I can't deploy mongodb

Reply:
![image](https://github.com/user-attachments/assets/0f32c413-4142-45bf-a0bc-077d058c6e10)

I could successfully deploy `mongodb` using URL `docker.io/library/mongo`. Some temporary/network problem?

Reply:
What version of the mongo DB image do you have? I have `a31b196b207d` which should be the latest image, since I recently deleted all my container images to free some space. Maybe this affects it?

Reply:
I'll double check on windows

Reply:
Can I check it with OS Tools somehow?

Docker says the `mongo:latest` is `a31b196b207d`...

Reply:
The command result is an error for me, since it detects windows and linux versions of the docker image

Reply:
> The command result is an error for me, since it detects windows and linux versions of the docker image

I didn't get this...  So, you have the same error result while deploying the same image on Windows and *nix? 
And what does it give to us? 

... And yes, I think if you hit the docker rate limit, you'll probably won't  be able to deploy any image till the counter is dropped. Not 100% sure though.

Reply:
Seems to be working now for me. Does it work for you, Victor?